File: backend/src/main/java/com/godeltech/mastery/backend/domain/dto/responce/ServiceOperationRecordDTO.java
package com.godeltech.mastery.backend.domain.dto.responce;

import java.time.LocalDate;
import java.util.List;

public record ServiceOperationRecordDTO(
  Long id,
  String serviceOperationNumber,
  LocalDate date,
  Integer mileage,
  List<ServiceWorkDTO> serviceWorks,
  List<ChangeablePartDTO> changableParts
) {}

File: backend/src/main/kotlin/com/godeltech/mastery/backend/domain/dto/responce/ServiceOperationRecordDto.kt
package com.godeltech.mastery.backend.domain.dto.responce

import java.time.LocalDate

data class ServiceOperationRecordDto(
    val id: Long?,
    val serviceOperationNumber: String?,
    val date: LocalDate?,
    val mileage: Int?,
    val serviceWorks: List<ServiceWorkDto>?,
    val changableParts: List<ChangeablePartDto>?
)
